packages/react-components/src/components/Attachment/AttachmentCard.tsx

```diff
@@ -124,6 +124,7 @@ export const _AttachmentCard = (props: _AttachmentCardProps): JSX.Element => {
                     imageFileType: 'svg'
                   }).iconName
                 }
+                role="img"
```

Comment:
Should this be on the parent div?

Reply:
good point, let me move it to parent div so we can be consistent